testpar/t_2Gio.c

```diff
@@ -1348,6 +1266,22 @@ dataset_writeAll(void)
     /* Dataset5: point selection in File - Hyperslab selection in Memory*/
     /* create a file dataspace independently */
     point_set(start, count, stride, block, num_points, coords, OUT_OF_ORDER);
+    if (VERBOSE_MED) {
```

Comment:
This previously existed in `point_set()`, but that function was moved into the parallel testing library where `VERBOSE_MED` isn't available (since some tests don't integrate with the testframe.c framework). So this output section was moved into individual tests that do have access to `VERBOSE_MED`.

Reply:
Will look into pulling that back out into a separate function that isn't tied to `VERBOSE_MED`